city_scrapers/spiders/newnj_nmc.py

```diff
@@ -0,0 +1,38 @@
+from city_scrapers_core.constants import CITY_COUNCIL
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import LegistarSpider
+
+
+class NewnjSpider(LegistarSpider):
+    name = "newnj_nmc"
+    agency = "Newark Municipal Council"
+    timezone = "America/Chicago"
```

Comment:
If I'm not mistaken, the time in Newark is 1 hour ahead of Chicago. Therefore, I think we should use `America/New_York` for this spider's attribute.

Reply:
You are completely correct! I have forgotten to change the timezone before. Thank you for catching this, @lamle-ea  🙏